pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -657,11 +659,9 @@ def newpkgcb(pkgId, name, arch):
 
             # now send modules down the pipeline since all relations have been set up
             for modulemd in modulemd_list:
-                modulemd_pb.increment()
```

Comment:
Modules and Comps are already counted way up above.  Also these ```increment()``` calls are outside of their context managers so they would be un-throttled and slow.